packages/diagrams/frontend/sirius-components-diagrams-reactflow/src/renderer/node/ConnectionSourceHandles.tsx

```diff
@@ -0,0 +1,260 @@
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ *
+ * Contributors:
+ *     Obeo - initial API and implementation
+ *******************************************************************************/
+import { gql, useQuery } from '@apollo/client';
+import { useMultiToast } from '@eclipse-sirius/sirius-components-core';
+import { Theme, useTheme } from '@material-ui/core/styles';
+import React, { useContext, useEffect, useState } from 'react';
+import { Handle, Position, useUpdateNodeInternals } from 'reactflow';
+import { DiagramContext } from '../../contexts/DiagramContext';
+import { DiagramContextValue } from '../../contexts/DiagramContext.types';
+import { ConnectorContext } from '../connector/ConnectorContext';
+import { ConnectorContextValue } from '../connector/ConnectorContext.types';
+import { useConnector } from '../connector/useConnector';
+import {
+  GQLDiagramDescription,
+  GQLGetToolSectionsData,
+  GQLGetToolSectionsVariables,
+  GQLNodeDescription,
+  GQLRepresentationDescription,
+  GQLSingleClickOnTwoDiagramElementsTool,
+  GQLTool,
+} from '../connector/useConnector.types';
+import { ConnectionSourceHandlesProps, ConnectionSourceHandlesStates } from './ConnectionSourceHandles.types';
+
+const getToolSectionsQuery = gql`
+  query getToolSections($editingContextId: ID!, $diagramId: ID!, $diagramElementId: ID!) {
+    viewer {
+      editingContext(editingContextId: $editingContextId) {
+        representation(representationId: $diagramId) {
+          description {
+            ... on DiagramDescription {
+              palette(diagramElementId: $diagramElementId) {
+                tools {
+                  __typename
+                  ... on SingleClickOnTwoDiagramElementsTool {
+                    candidates {
+                      sources {
+                        id
+                      }
+                      targets {
+                        id
+                      }
+                    }
+                  }
+                }
+                toolSections {
+                  tools {
+                    __typename
+                    ... on SingleClickOnTwoDiagramElementsTool {
+                      candidates {
+                        sources {
+                          id
+                        }
+                        targets {
+                          id
+                        }
+                      }
+                    }
+                  }
+                }
+              }
+            }
+          }
+        }
+      }
+    }
+  }
+`;
+
+const customConnectionHandlesStyle = (position: Position, theme: Theme): React.CSSProperties => {
+  switch (position) {
+    case Position.Right:
+      return {
+        position: 'absolute',
+        borderRadius: '0',
+        right: '-15px',
+        border: 'solid black',
+        borderWidth: '0 2px 2px 0',
+        display: 'inline-block',
+        padding: '2px',
+        transform: 'rotate(-45deg)',
+        WebkitTransform: 'rotate(-45deg)',
```

Comment:
Is the Webkit-specific property really needed? I don't see any other uses elsewhere in our code. On the contrary, we have several places where we use the standard `transform:` and do not add a Webkit-specific variant.